rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3ServerHandler.java

```diff
@@ -58,10 +67,21 @@ protected void channelRead0(ChannelHandlerContext ctx, ByteBufHolder request) th
         ByteBuf responseContent = Unpooled.buffer();
         int responseCode;
         try (ByteBufOutputStream os = new ByteBufOutputStream(responseContent);
-             ByteBufInputStream is = new ByteBufInputStream(request.content().retain())){
-            
+             ByteBufInputStream is = new ByteBufInputStream(request.content().retain())) {
+
             responseCode = jsonRpcServer.handleRequest(is, os);
-        } catch (Exception e) {
+        } catch (IllegalArgumentException e) {
```

Comment:
This exception is quite broad, it could (maybe) be thrown due to errors other than the ones in the new validator. I would catch a custom exception, see related comment here.

Reply:
Done